### PR TITLE
doc: add minute for 2024-09-11

### PR DESCRIPTION
**Pull request by marco-ippolito**
Fixes: https://github.com/nodejs/typescript/issues/8
@nodejs/typescript

### CODE REVIEW

File: meetings/2024-09-11.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+Marco: work going on, some issues on windows arm64 and a PR for typescript support in module mocking.
+* Import specifiers in `--experimental-strip-types` [#214](https://github.com/nodejs/loaders/issues/214)
+Agreement that node should not perform extension guessing, TypeScript team is working on a module resolution compatible.
+Discussion on edge cases of module resolution such as importing .ts files from .js files.
+Marco: currently untested but working in Node.js
+Daniel: Where could we test edge cases in node.js?
+Marco: add test/es-module/test-typescript.mjs in the node repository
+Andrew: working on a module resolution for node compatibility but there are some edge cases, so need to figure out how handle those (errors, warning etc…)
+Marco: should we move the meeting once a month?
+Daniel: lets keep the meeting twice a month and cancel if no agenda
```
Comment (JakobJingleheimer) [inline meetings/2024-09-11.md:37]:
```suggestion
* Import specifiers in `--experimental-strip-types` [#214](https://github.com/nodejs/loaders/issues/214)
    Agreement that node should not perform extension guessing, TypeScript team is working on a module resolution compatible.
    Discussion on edge cases of module resolution such as importing .ts files from .js files.
    Marco: currently untested but working in Node.js
    Daniel: Where could we test edge cases in node.js?
    Marco: add test/es-module/test-typescript.mjs in the node repository
    Andrew: working on a module resolution for node compatibility but there are some edge cases, so need to figure out how handle those (errors, warning etc…)
    Marco: should we move the meeting once a month?
    Daniel: lets keep the meeting twice a month and cancel if no agenda
```

File: meetings/2024-09-11.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+## Q&A, Other
```
Comment (JakobJingleheimer) [inline meetings/2024-09-11.md:39]:
```suggestion
## Q&A, Other

The codemode to fix specifiers in existing TypeScript projects is nearly ready for user testing; confirmed against happy-paths, monorepos (npm & yarn, but not pnpm, which behaves in a non-standard way). Testing again multiple real-world projects completed successfully. Now testing against Puppeteer's repo.
```
